plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -57,38 +57,33 @@ void
 TimingTriggerCandidateMaker::do_conf(const nlohmann::json& config)
 {
   auto params = config.get<dunedaq::trigger::timingtriggercandidatemaker::Conf>();
-  //			try {
-  m_map.push_back({ params.s0.time_before, params.s0.time_after });
-  m_map.push_back({ params.s1.time_before, params.s1.time_after });
-  m_map.push_back({ params.s2.time_before, params.s2.time_after });
-  //			} catch(...)  {
-  //				ERS_LOG(get_name() << " unsuccessfully configured");
-  //			}
-  //			ERS_LOG(get_name() << " successfully configured");
+  try {
```

Comment:
I don't think there's anything in this block that can throw an `ers::Issue`, is there? If there's no chance of throwing, the try/catch block should be removed (or changed to catch an exception that _can_ actually be thrown here)

Reply:
ok